addons/account/models/account_journal_dashboard.py

```diff
@@ -78,14 +78,13 @@ def _query_has_sequence_holes(self):
         self.env.cr.execute("""
             SELECT move.journal_id,
                    move.sequence_prefix
-              FROM account_move move
-             WHERE move.journal_id = ANY(%(journal_ids)s)
+              FROM account_move move, account_journal journal
+             WHERE (journal.type = 'sale' OR journal.type = 'purchase')
```

Comment:
```suggestion
             WHERE journal.type IN ('sale', 'purchase')
```
I guess you need to do both the filtering on ids and journal type as only favorite journals appear in the journal dashboard.